test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -366,8 +366,8 @@ propLockstep_RealImpl_MockFS_IO tr cleanupFlag fsFlag refsFlag =
 -- We can not use @bracket@ inside @PropertyM@, so @acquire_RealImpl_MockFS@ and
 -- @release_RealImpl_MockFS@ are not run in a masked state and it is not
 -- guaranteed that the latter runs if the former succeeded. Therefore, if
--- @runActions@ fails (with exceptions), then not having @bracket@ might lead to
--- more exceptions, which can obfuscate the orginal reason that the property
+-- @runActions@ fails (with exceptions), then not hasving @bracket@ might lead to
```

Comment:
I don't know if "having" to "hasving" is really that much of an improvement.

Reply:
Fixed.